frontend/tests/features/wheresMyGene/landingPage.ts

```diff
@@ -0,0 +1,104 @@
+import { expect, Page, test } from "@playwright/test";
+import { TEST_URL } from "../../common/constants";
+import { ROUTES } from "src/common/constants/routes";
+import {
+  ADD_GENE_BTN,
+  ADD_GENE_LBL,
+  ADD_TISSUE_LBL,
+} from "tests/utils/constants";
+
+const { describe } = test;
+const ALERT =
+  "We would appreciate your feedback, please fill out a quick survey";
+
+const SURVEY_LINK = "https://airtable.com/shrLwepDSEX1HI6bo";
+const EXPLORE_GENE_EXPRESSION = "explore-gene-expression";
+const LEGEND_WRAPPER = "legend-wrapper";
+const DOT_SIZES: Array<number> = [4, 9, 12, 14, 16];
+async function checkDotSize(size: number, index: number, page: Page) {
+  expect(
+    page
+      .locator('[id="expressed-in-cells-dots"]')
+      .locator("span")
+      .nth(index)
+      .getAttribute("size")
+  ).toBe(DOT_SIZES[size]);
+}
```

Comment:
discussion:

`page.locator('[id="expressed-in-cells-dots"]').locator("span")` matches all 5 dots already. Maybe we can convert the matches into an array of sizes and then do:

```ts
const actualSizes = page.locator('[id="expressed-in-cells-dots"]').locator("span").getAttribute("size")

expect(actualSizes).toEqual(DOT_SIZES)
```

Reply:
Oh I see what you mean; matching whole array instead of loop?

Reply:
Yeah I'm just thinking that way we don't have to query the page N times in a loop, since we already have all the information needed in the first time!